### PR TITLE
Update "dev" profile for sdk/spring/pom.xml:

### PR DESCRIPTION
**Pull request by chenrujun**
Update "dev" profile for sdk/spring/pom.xml:
1. Add a new module: "./boms/spring-cloud-azure-dependencies".
2. Add description about how to install all Spring Cloud Azure artifacts quickly.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @saragluna , @backwind1233 . Please help to review this PR when you have time.

File: sdk/spring/pom.xml [hunk 1]
```diff
@@ -194,9 +194,27 @@
...
+              -o \
+              -Dcheckstyle.skip \
+              -Dcodesnippet.skip \
+              -Denforcer.skip \
+              -Djacoco.skip \
+              -Dmaven.javadoc.skip \
+              -Dmaven.test.skip \
+              -Drevapi.skip \
+              -Dspotbugs.skip
+       -->
```
Comment (saragluna) [inline sdk/spring/pom.xml:213]: we'd better add this to the README

Comment (chenrujun) [reply sdk/spring/pom.xml]: Updated.

File: sdk/spring/README.md [hunk 2]
```diff
@@ -15,10 +15,22 @@ cd azure-sdk-for-java
...
+  -T 4 \
+  -o \
+  -Dcheckstyle.skip \
+  -Dcodesnippet.skip \
+  -Denforcer.skip \
+  -Djacoco.skip \
+  -Dmaven.javadoc.skip \
+  -Dmaven.test.skip \
+  -Drevapi.skip \
+  -Dspotbugs.skip
```
Comment (saragluna) [inline sdk/spring/README.md:33]: can this run in powershell?

Comment (chenrujun) [reply sdk/spring/README.md:33]: Added another command for PowerShell.

**Review by saragluna — APPROVED** [review]
LGTM.

File: sdk/spring/README.md [hunk 3]
```diff
@@ -15,10 +15,27 @@ cd azure-sdk-for-java
...
+```
+
+If you are using PowerShell, please use the following command instead:
+```shell
+mvn clean install -f sdk/spring/ -Pdev -T 4 -o "-Dcheckstyle.skip" "-Dcodesnippet.skip" "-Denforcer.skip" "-Djacoco.skip" "-Dmaven.javadoc.skip" "-Dmaven.test.skip" "-Drevapi.skip" "-Dspotbugs.skip"
```
Comment (backwind1233) [inline sdk/spring/README.md:38]: in powershell you can use `

Comment (chenrujun) [reply sdk/spring/README.md:38]: image

**Pull request merged**